src/components/MoneyReportHeader.tsx

```diff
@@ -202,8 +202,9 @@ function MoneyReportHeader({policy, report: moneyRequestReport, transactionThrea
 
         const anyTransactionOnHold = selectedTransactions.some(isOnHoldTransactionUtils);
         const allTransactionOnHold = selectedTransactions.every(isOnHoldTransactionUtils);
+        const isReportApprovedAndPaid = moneyRequestReport?.stateNum === CONST.REPORT.STATE_NUM.APPROVED && moneyRequestReport?.statusNum === CONST.REPORT.STATUS_NUM.REIMBURSED;
```

Comment:
I think we just name this paid. You can use the `isReportManuallyReimbursed` method for that too
```suggestion
        const isReportReimbursed = moneyRequestReport?.stateNum === CONST.REPORT.STATE_NUM.APPROVED && moneyRequestReport?.statusNum === CONST.REPORT.STATUS_NUM.REIMBURSED;
```